src/headers/game.hpp

```diff
@@ -55,6 +55,8 @@ class Game {
   ull score;
   ull bestScore;
   ull largestTile;
+  int cellWidth;
+  std::string horizontalBorder;
```

Comment:
Better practice would be:
* A free anonymous-namespace function that outputs a horizontalBorder string.